src/main/scala/com/scylladb/migrator/Validator.scala

```diff
@@ -139,10 +139,63 @@ object Validator {
 
     val failures = runValidation(migratorConfig)
 
-    if (failures.isEmpty) log.info("No comparison failures found - enjoy your day!")
-    else {
+    if (failures.isEmpty) {
```

Comment:
don't forget that 
config.validation.failuresToFetch
will limit above failures
so figuring out proper https://github.com/scylladb/scylla-migrator/blob/master/config.yaml.example#L217 is the limit for this list